deploy/stacks/backend_stack.py

```diff
@@ -146,12 +155,18 @@ def __init__(
             envname=envname,
             resource_prefix=resource_prefix,
             vpc=vpc,
-            vpc_endpoints_sg=vpc_endpoints_sg,
+            vpce_connection=vpce_connection,
             ecr_repository=repo,
             image_tag=image_tag,
             prod_sizing=prod_sizing,
             pivot_role_name=self.pivot_role_name,
             tooling_account_id=tooling_account_id,
+            s3_prefix_list=self.s3_prefix_list,
```

Comment:
NIce, since there are different sg for each lambda now we can ser all sg here, but in aurora only the api and aws handlers will get access